### PR TITLE
update OpenStack CSI & cloud-controller manifest

### PR DESCRIPTION
**Pull request by zetaab**

### CODE REVIEW

**Comment by zetaab** [comment]
@hakman / @olemarkus do you know do we have any way to delete old resources? Basically I should execute following to migrate to deployment

```
kubectl delete role external-resizer-cfg -n kube-system
kubectl delete rolebinding csi-resizer-role-cfg -n kube-system
kubectl delete sts csi-cinder-controllerplugin -n kube-system
```

File: upup/models/cloudup/resources/addons/storage-openstack.addons.k8s.io/k8s-1.16.yaml.template [hunk 1]
```diff
@@ -440,7 +425,7 @@ spec:
       hostNetwork: true
       containers:
         - name: node-driver-registrar
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.1.0
```
Comment (olemarkus) [inline upup/models/cloudup/resources/addons/storage-openstack.addons.k8s.io/k8s-1.16.yaml.template:443]: registry.k8s.io supersedes k8s.gcr.io, so we probably should not change this.

**Review by olemarkus — CHANGES_REQUESTED** [review]

File: upup/models/cloudup/resources/addons/storage-openstack.addons.k8s.io/k8s-1.16.yaml.template (hunk 1 above)
Comment (zetaab) [reply upup/models/cloudup/resources/addons/storage-openstack.addons.k8s.io/k8s-1.16.yaml.template:443]: ok I was just copying values from openstack manifests.. I can change these back and verify that images exists

**Comment by zetaab** [comment]
@olemarkus changed all back to registry.k8s.io and verified that all images do exist

**Review by olemarkus — APPROVED** [review]

**Pull request merged**